src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -131,7 +131,7 @@ private Task AppendProjectDiagnosticsAsync(Project project, CancellationToken ca
                     return SpecializedTasks.EmptyTask;
                 }
 
-                return AppendProjectAndDocumentDiagnosticsAsync(project, d => d.ProjectId == project.Id, cancellationToken);
+                return AppendProjectAndDocumentDiagnosticsAsync(project, null, d => d.ProjectId == project.Id, cancellationToken);
```

Comment:
This is code that I had to touch incidentally to fix a perf problem. This file is ripe with method calls that would benefit from use of named args. However, I would rather not do this as part of the current change :)